packages/asc-ui/src/components/Select/Select.tsx

```diff
@@ -24,37 +24,25 @@ const Select = React.forwardRef<
     const srOnly = srOnlyProp || false
 
     return (
-      <>
-        {label && (
-          <FormLabelStyle
-            srOnly={srOnly}
-            htmlFor={id}
-            label={label}
-            style={labelStyle}
-          >
-            {label}
-          </FormLabelStyle>
-        )}
-        {errorMessage && (
-          <FormLabelStyle
-            srOnly={srOnly}
-            htmlFor={id}
-            error={errorMessage}
-            style={errorStyle}
-          >
-            {errorMessage}
-          </FormLabelStyle>
-        )}
+      <FormLabelStyle
+        htmlFor={id}
+        srOnly={srOnly}
+        label={label}
+        error={errorMessage}
+        style={errorMessage ? errorStyle : labelStyle}
+      >
+        {errorMessage || label}
```

Comment:
Probably out of scope for this PR but error messages should really not hijack a label element in order to display an error message. I've created #359 to address this problem.

Reply:
True, thanks!